Applications/CMC/cmc.cpp

```diff
@@ -71,6 +72,11 @@ int main(int argc,char **argv)
             return(0);
  
         // PRINT A DEFAULT SETUP FILE FOR THIS INVESTIGATION
+        } else if(startsWith("-j",option.c_str())){
+            char* optionCStr = const_cast<char*>(option.c_str());
+            optionCStr += 2; //increment the char*'s memory address by 2, skip "-j"
+            specifiedMaxNumThreads = atoi(optionCStr);
+
```

Comment:
It's cool. I really like the idea of using a library to do the parsing vs doing it manually. :+1: